content/swiftui/concepts/views/terms/table/table.md

```diff
@@ -0,0 +1,61 @@
+---
+Title: SwiftUI table
+Description: Table view in SwiftUI
+Subjects: Mobile Development, iOS application, SwiftUI
+Tags: SwiftUI, iOS, table, view
+
+---
+
+# SWIFTUI - LIST
+
+## **A container that presents rows of data arranged in one or more columns, optionally providing the ability to select one or more members**
+
+You commonly create tables from collections of data.
+If there are more rows than can fit in the available space,  table provides vertical scrolling automatically (on macOS, the table also provides horizontal scrolling if there are more columns than can fit in the width of the view)
+
+>struct Product: Identifiable {
+ let id: UUID()
+ let name: String
+ let color: String
+ let price: Double
+}
+
+>@State private var products = [
+	Product(name: "Pen", color: "Black", price: 2.99),
+	Product(name: "Eraser", color: "White", price: 1.49),
+	Product(name: Pencil, color: "Green", price: 0.99)
+	]
+
+>struct ProductTable: View {
+	var body: some View {
+		Table (products) {
+			TableColumn ("Product Name", value: \.name)
+			TableColumn("Product Color", value: \.color)
+			TableColumn("Product Price", value: \.price) 
+		}
+	 }
+	}
+	
+ 
+
+
+
+### You interact with the table by selecting one or more rows and then performing actions from the toolbar or a context menu.
```

Comment:
```suggestion
Tables are interacted with by selecting one or more rows and then performing actions from the toolbar or a context menu.
```
This isn't appropriate for a header. Headers are 5 words or less, in title case, and have no ending punctuation.